app/models/ValidationError.scala

```diff
@@ -38,3 +38,18 @@ final case class MismatchedPage(query: Query) extends ValidationError {
 
   val message: String = s"Mismatched page: ${query.path}"
 }
+
+final case class UnexpectedNoExemption(query: Query) extends ValidationError {
+
+  val message: String =
+    s"Assessment page has answer No Exemption but subsequent pages have been answered: ${query.path}"
+}
+
+final case class MissingAssessmentAnswers(query: Query) extends ValidationError {
```

Comment:
is this one still relevant?

Reply:
Still used in the "ensureHaveAnsweredTheRightAmount" check